docs/source/reference/datasets.rst

```diff
@@ -3,9 +3,9 @@ Datasets
 
 .. module:: chainercv.datasets
 
-DirectoryParsingClassificationDataset
+DirectoryParsingLabelDataset
 -------------------------------------
```

Comment:
Sorry. This line should also be fixed. (the length of `-`)

Reply:
Oh. Sorry...